cinema/gamemode/localization/pirate.lua

```diff
@@ -19,21 +19,21 @@ LANG.Set                        = "Set"
 -- modules/theater/sh_theater.lua
 LANG.Theater_VideoRequestedBy       = C("Commander of this ship be ",ColHighlight,"%s",ColDefault,".")
 LANG.Theater_InvalidRequest         = "Invalid video request."
-LANG.Theater_AlreadyQueued          = "The requested ship is already in the fleet."
+LANG.Theater_AlreadyQueued          = "The requested ship is already in harbour."
 LANG.Theater_ProcessingRequest      = C("Processing ",ColHighlight,"%s",ColDefault," request...")
-LANG.Theater_RequestFailed          = "The requested ship has sunk."
-LANG.Theater_Voteskipped            = "The ship has been taken over by rebellious crew."
-LANG.Theater_ForceSkipped           = C(ColHighlight,"%s",ColDefault," has seized ownership of thar vessel.")
-LANG.Theater_PlayerReset            = C(ColHighlight,"%s",ColDefault," has swabbed the poopdeck.")
-LANG.Theater_LostOwnership          = "You have lost your ship due to jumping overboard."
-LANG.Theater_NotifyOwnership        = "You're now the owner of the ship."
-LANG.Theater_OwnerLockedQueue       = "The captain has closed harbor."
-LANG.Theater_LockedQueue            = C(ColHighlight,"%s",ColDefault," has closed the fleet to new ships.")
-LANG.Theater_UnlockedQueue          = C(ColHighlight,"%s",ColDefault," has opened the fleet to new ships.")
+LANG.Theater_RequestFailed          = "The requested ship sank."
+LANG.Theater_Voteskipped            = "The ship has been taken over by rebellious crew!"
+LANG.Theater_ForceSkipped           = C(ColHighlight,"%s",ColDefault," has sank the vessel.")
+LANG.Theater_PlayerReset            = C(ColHighlight,"%s",ColDefault," has swabbed the poopdeck!")
+LANG.Theater_LostOwnership          = "You have lost command of the harbour due to drowning."
```

Comment:
This file needs quite a bit of you -> ye imo, for example "Ye've lost command of yer harbour!"
This line specifically needs a reword, perhaps what I suggested above.